packages/paste-core/components/in-page-navigation/src/InPageNavigation.tsx

```diff
@@ -95,20 +145,31 @@ const InPageNavigation = React.forwardRef<HTMLDivElement, InPageNavigationProps>
       <InPageNavigationContext.Provider value={{ variant, orientation }}>
         <Box {...safelySpreadBoxProps(props)} as="nav" ref={ref} element={element}>
           <Box
-            as="ul"
-            listStyleType="none"
-            element={`${element}_ITEMS`}
+            as={StyledScrollTabList as any}
```

Comment:
suggestion(blocking): let's keep this a `ul` (was this change intentional?)

Reply:
No this was probably just a copy paste error. I updated the styled component to stay `ul`. Good catch!

Reply:
Actually I reverted that commit. It looks like I removed that `ul` but I actually just wrapped it. The original box is now just nested